antsibull/build_ansible_commands.py

```diff
@@ -300,7 +318,7 @@ async def make_collection_dist(name, version, package_dir, dest_dir):
     await loop.run_in_executor(None, shutil.move, dist_file, dest_dir)
 
 
-async def make_collection_dists(dest_dir, collection_dirs):
+async def make_collection_dists(dest_dir: str, collection_dirs: t.List[str]):
```

Comment:
```suggestion
async def make_collection_dists(dest_dir: str, collection_dirs: t.List[str]) -> None:
```

Reply:
Fixed in 6645183c0926dd6758bcc4c2133bdea67efc90a3 (found a few more).